### DISCUSSION TITLE
Dynamic subdomains on vercel.app domain

### DISCUSSION THREAD

**Discussion by thedavedavies** [vercel/community#1695] [Help] [answered]
### Summary

I've set up dynamic subdomains using the [platforms](https://github.com/vercel/platforms/blob/main/middleware.ts) middleware method. This works great on localhost, and also when setting up a live custom domain name using a wildcard SSL ( ie. **_*.custom-domain.com_**).

However, when I want to test the app deployment on ***.my-test-domain.vercel.app** before deployment I get an SSL error. I know that this is because vercel doesn't allow wildcard SSLs - but is there a way I can set up an exception for myself so I can test the build on Vercel before deploying?

**Comment by amyegan** [comment]
Hi @thedavedavies! You can set up a wildcard with your account slug, like ~`*.<username>.vercel.com`~

Edit: I had the wrong example URL. 🙈 I should be `*.<username>.vercel.app`

**Reply by thedavedavies** [reply]
Is that on `https://vercel.com/<username>/subdomains/settings/domains`?  I've tried adding `*.<username>.vercel.com` (replacing `<username>` with my own username) as a domain in my project setting, but it tells me:

_**Another Vercel account is using this domain. Please set the following TXT record on `_vercel.vercel.com` to use `*.<username>.vercel.com` in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.**_

**Reply by smaeda-ks** [reply, marked as answer]
@thedavedavies `*.<username>.vercel.com` -> `*.<username>.vercel.app`, could you try this instead?

**Reply by thedavedavies** [reply]
Thanks - this works great.
